.gitignore

```diff
@@ -17,3 +17,4 @@ build/
 coverage.xml
 jupyter_execute/
 venv/*
+.aider*
```

Comment:
What is this new addition related to?

Reply:
Ah, that was some VSCode extension I was experimenting with. I removed that from the `.gitignore` file.